dhall-docs/src/Dhall/Docs/Html.hs

```diff
@@ -66,6 +78,18 @@ exprToHtml expr = renderTree prettyTree
             Builtin -> "builtin"
             Operator -> "operator"
 
+    typeLayout :: Pretty.Doc ann -> Pretty.SimpleDocStream ann
+    typeLayout = Pretty.removeTrailingWhitespace . Pretty.layoutSmart opts
+      where
+        -- this is done so the type of a dhall file fits in a single line
+        -- its a safe value, since types in source codes are not that large
```

Comment:
I think this comment is a bit outdated now. At the least the bit regarding the safe vaue.